MathTrainer/MathTrainer/TrainViewController.swift

```diff
@@ -14,5 +14,26 @@ final class TrainViewController: UIViewController {
             print(type)
         }
     }
+    
+    // MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureButtons2()
+        
+    }
+    
+    // MARK: - IBOutlets
+    @IBOutlet var buttonCollection2: [UIButton]!
```

Comment:
IBOutlets must be before life cycle methods, where you have `var type: MathTypes = .add`